lib/repo/repository_test.go

```diff
@@ -1,43 +1,56 @@
 package repo
 
 import (
-	"strings"
-	"sync"
+	"os"
 	"testing"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
__[gometalinter]__
* [gosimple] could not import github.com/stretchr/testify/assert (cannot find package "github.com/stretchr/testify/assert" in any of:
* [staticcheck] could not import github.com/stretchr/testify/assert (cannot find package "github.com/stretchr/testify/assert" in any of:

<!-- Comment by [SideCI](https://sideci.com) -->